### ISSUE TITLE
Dataset validation <gml:descriptionReference> empty value

### ISSUE THREAD

**Issue by Dnoack49** [INSPIRE-MIF/helpdesk-validator#787]
Dear Validator Team,

I validated the following Dataset via the Option "**Remote File**":
https://inspire.brandenburg.de/services/br_natraum_wfs?service=WFS&version=2.0.0&request=GetFeature&typenames=br:Bio-geographicalRegion&Count=10 

for which an error regarding the **Conformance class: INSPIRE GML application schemas, General requirements** (gmlas.c.1:Consistency /GML model) is thrown:
![grafik](https://user-images.githubusercontent.com/8046960/173861807-5e6bbe69-204a-4124-b9d8-afe98cc99a13.png)

The error message received is:
![grafik](https://user-images.githubusercontent.com/8046960/173858243-481cc8af-60e7-4170-a2c8-edb2115cbf9c.png)

Here the Test report: 
[Remote File - test suite Annex III - Bio-geographical regions (BR).zip](https://github.com/INSPIRE-MIF/helpdesk-validator/files/8910611/Remote.File.-.test.suite.Annex.III.-.Bio-geographical.regions.BR.zip)

The descriptionReference-element in the gml looks like this:
![grafik](https://user-images.githubusercontent.com/8046960/173858913-ae9621c3-97b2-4164-b4d7-5005146f293f.png)

However, when I validate the same dataset via the Option "File upload" (using the gml which results from the GetFeature-Request above):
[br_natraum.zip](https://github.com/INSPIRE-MIF/helpdesk-validator/files/8910575/br_natraum.zip)

The Test passes:
![grafik](https://user-images.githubusercontent.com/8046960/173862500-3a49a2b4-ba44-478f-baed-cc3b2a583114.png)

Here the Test report: 
[File Upload - test suite Annex III - Bio-geographical regions (BR).zip](https://github.com/INSPIRE-MIF/helpdesk-validator/files/8910637/File.Upload.-.test.suite.Annex.III.-.Bio-geographical.regions.BR.zip)

Which result is correct?

Thank you.

**Comment by fabiovinci** [comment]
Dear @Dnoack49,

I checked your datasets and I noticed that there are two different encodings for the same element "gml:descriptionReference".
In the WFS response the `xlink:href` is missing and this generates the error you reported.

`<gml:descriptionReference xmlns:xlink="http://www.w3.org/1999/xlink" xlink:title="Naturräumliche Regionen des Landschaftsprogramms u.a. auf der Grundlage der 'Naturräumlichen Gliederung Brandenburgs (nach Scholz, 1962)', zur Verwendung im Rahmen des Landschaftsprogramms"/>`

According to the different encoding, both results are correct.
